AES.Entities/Migrations/AESDbConfiguration.cs

```diff
@@ -132,7 +132,21 @@ protected override void Seed(AESDbContext context)
                 Zip = 97219
             };
 
+            var userInfo2 = new UserInfo()
+            {
+                CallEndTime = new TimeSpan(10, 0, 0),
```

Comment:
Since we work later than this, can you make it 0001 - 2359? Just to ensure that there's always something there.